api_contracts/contracts/downloadCenter/DownloadCenter.md

```diff
@@ -53,7 +57,33 @@ This endpoint returns a list of budget subfunctions with their associated title
 
 + Response 200 (application/json)
     + Attributes
-        + results (BudgetSubfunctionResults, fixed-type) 
+        + results (required, array[BudgetSubfunction], fixed-type)
+
+## Agency Function [/api/v2/bulk_download/list_agencies/]
```

Comment:
Not sure why these have `Function` in the name - something like `List Agencies`, `Federal Accounts by Agency` may be more clear